EKF/RingBuffer.h

```diff
@@ -127,7 +127,7 @@ class RingBuffer {
 			int index = (_head - i);
 			index = index < 0 ? _size + index : index;
 
-			if (timestamp >= _buffer[index].time_us && timestamp - _buffer[index].time_us < (uint64_t)1e5) {
+			if (timestamp >= _buffer[index].time_us && timestamp < _buffer[index].time_us + (uint64_t)1e5) {
```

Comment:
Why half a second?

Reply:
It is 0.1 seconds.
But I don't know why.

Reply:
Oops, confused with 5e5 that we use in a few other places.

Reply:
It would be nice to document if we get an answer, unrelated to the potential change/fix though.